### PR TITLE
Issue #13097: Split sizes xdocs into separate pages

### PR DESCRIPTION
**Pull request by stoyanK7**
Part of #13097

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1984,6 +1984,7 @@
             <excludedLink>https://paypal.com/</excludedLink>
             <!-- temporal suppress till plugin update link in his source -->
             <excludedLink>http://www.mojohaus.org/exec-maven-plugin</excludedLink>
+            <excludedLink>http://www.mojohaus.org/xml-maven-plugin</excludedLink>
```
Comment (stoyanK7) [inline pom.xml:1987]: This one gives 301. I noticed similar links were present in `pom.xml` so I placed it next to them.
```
------------ grep of linkcheck.html--BEGIN
1a2
> <td><i><a class="externalLink" href="http://www.mojohaus.org/xml-maven-plugin/">http://www.mojohaus.org/xml-maven-plugin/</a>: 301 Moved Permanently</i></td></tr></table></td></tr>
------------ grep of linkcheck.html--END
Exit code:1

```

Comment (stoyanK7) [reply pom.xml:1987]: Moving to a separate PR to merge faster.

Comment (stoyanK7) [reply pom.xml:1987]: Resolved.

**Comment by romani** [comment]
Rebased.

**Review by romani — APPROVED** [review]
Ok to merge

**Comment by stoyanK7** [comment]
Fixed descriptions.

**Comment by nrmancuso** [comment]
Github, generate site

**Review by nrmancuso — APPROVED** [review]
Ok to merge after conflicts are resolved.

**Comment by stoyanK7** [comment]
@nrmancuso Conflicts are resolved.

**Comment by romani** [comment]
@stoyanK7 , conflict.

**Comment by stoyanK7** [comment]
@romani Fixed.

**Pull request merged**